tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -6036,27 +6037,35 @@ Status ConvertResize(OpConverterParams* params) {
                                  node_def.name());
   }
 
+  // Validate inputs_tensor.
+  // TODO: Remove when Dynamic Shape will be supported for this layer.
```

Comment:
Can we change this TODO to:
TODO: Allow dynamic shape for input-1 when we can handle shape input tensors.